Dockerfile

```diff
@@ -7,31 +7,35 @@ ENV NODE_OPTIONS="--max-old-space-size=4096"
 ARG VITE_APP_METABASE_INSTANCE_URL
 ENV VITE_APP_METABASE_INSTANCE_URL=${VITE_APP_METABASE_INSTANCE_URL}
 
-ARG VITE_APP_API_HOST
-ENV VITE_APP_API_HOST=${VITE_APP_API_HOST}
-
+ARG VITE_APP_BACKEND_HOST
+ENV VITE_APP_BACKEND_HOST=${VITE_APP_BACKEND_HOST}
 
 ARG WATCH=false
 ENV WATCH=${WATCH}
 
 WORKDIR /app
 
-COPY --exclude=./api --exclude=./metabase . .
+# Copy package files first (changes less frequently)
+COPY package.json yarn.lock ./
 
+# Install dependencies (cached unless package files change)
 RUN yarn --frozen-lockfile
 
+# Copy source code last (changes most frequently)
+COPY --exclude=./api --exclude=./metabase . .
+
 RUN if [ -d "./local-dist/embedding-sdk" ]; then \
-      echo "Local embedding-sdk dist is found in ./local-dist/embedding-sdk, installing it..."; \
-      yarn add file:./local-dist/embedding-sdk; \
-    else \
-      echo "Local embedding-sdk dist is not found in ./local-dist/embedding-sdk, skipping copy"; \
-    fi
+  echo "Local embedding-sdk dist is found in ./local-dist/embedding-sdk, installing it..."; \
```

Comment:
why these changes? auto-formatter?
I think the version on the left was better

Reply:
yeah, my editor did this :( Dang. I gotta fix this manually. Thanks for flagging this.

Reply:
fixed in 32fc609